### PR TITLE
Refactor: Updated links and styles.

### PR DESCRIPTION
**Pull request by shuntzhere**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
-        <div class="nav-logo flex justify-center">gehraaiyaan</div>
+        <div class="nav-logo flex justify-center">
+          <a href="./index.html"> gehraaiyaan </a>
+        </div>
         <div class="nav-right">
-          <button class="btn">
+          <button class="btn primary-btn">
             <p>Signup</p>
             <div>
               <img class="svg" src="/assets/svg/angle-right-solid.svg" />
```
Comment (nishasen) [inline index.html:20]: add alt attribute to img

**Review by nishasen — APPROVED** [review]
Good to merge, suggested a small change

File: index.html (hunk 1 above)
Comment (shuntzhere) [reply index.html:20]: That is an SVG I've added locally and so there is no alt text.

**Review by Sameony — APPROVED** [review]
cleanly ordered and structured code

**Pull request merged**